src/mito2/src/sst/parquet.rs

```diff
@@ -399,4 +411,115 @@ mod tests {
         let mut reader = builder.build().await.unwrap();
         check_reader_result(&mut reader, &[new_batch_by_range(&["b", "h"], 150, 200)]).await;
     }
+
+    fn customize_column_config(
```

Comment:
this method is copied from  [writer.rs](https://github.com/etolbakov/greptimedb/blob/c1e005b148cdfcaf55805cb69503ad9c70abb522/src/mito2/src/sst/parquet/writer.rs#L146). Will it be fine to leave it?

Reply:
As for tests, maybe we don't need to customize these properties for simplicity.